shopfloor_mobile/static/wms/src/scenario/scan_anything.js

```diff
@@ -25,10 +50,10 @@ Vue.component('scan-anything', {
                         this.reset_erp_data('data')
                     },
                     on_scan: (scanned) => {
-                        this.go_state(
-                            'wait_call',
-                            this.odoo.scan_anything(scanned.text)
-                        )
+                        this.$router.push({
```

Comment:
is this how the  switch to the new state should be handled now? Because then I think we should do it via `go_state`.